cypress/e2e/utils/sharedTests/loopTabsWithNewExpection.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  newClickFlow,
+  newExpectation,
+  newExpectationWithClickFlows,
+} from "../helpers";
+import { Expectation } from "../types";
+import { beVisible } from "./sharedFunctionsAndVariables";
+
+export type TabParams = {
+  test: string;
+  data_cy: string;
+  find: string;
+  text?: string;
```

Comment:
Some of these names could be clearer. Maybe something like:
```suggestion
  tabName: string;
  tabDataCy: string;
  dataCyToFind: string;
  textToFind?: string;
```